source/includes/_accounts.md

```diff
@@ -201,14 +300,85 @@ curl "https://app.sixpark.com.au/api/v1/users/:user_id/accounts/:account_id"
 
 ```json
 {
-  "id": "5cbcb043-4e20-46d6-9d85-71018debad6c",
-  "display_name": "FirstName LastName",
-  "open": false,
-  "links": {
-    "portfolio": "https://...",
-    "user": "https://...",
-    "self": "https://..."
-  }
+  "data": {
+    "id": "5cbcb043-4e20-46d6-9d85-71018debad6c",
+    "display_name": "FirstName LastName",
+    "strategy_name": "Balanced",
+    "relationships": {
+      "portfolio": {
+        "data": {
+          "id": "5326fc80-f144-44c6-839b-07f4a911de12",
+          "type": "portfolio"
+        }
+      }
+    },
+    "links": {
+      "portfolio": "https://...",
+      "user": "https://...",
+      "self": "https://..."
+    }
+  },
+  "included": [
+    {
+      "id": "ed83e246-c1ce-499a-9ed7-28c38fac1e7a",
+      "type": "holdings_total",
+      "attributes": {
+        "value": 868151.31,
```

Comment:
Where do you think the documentation for these attributes will exist?

Reply:
I might looks at doing it in the `_holdings_totals` page itself, and have an intro that details the full model properties, similar to https://woocommerce.github.io/woocommerce-rest-api-docs/#coupon-properties.

Would be handy to do that for holdings as well, given they don't have an API of their own.